Source/WebCore/Modules/mediastream/libwebrtc/LibWebRTCRtpTransformableFrame.cpp

```diff
@@ -138,6 +138,47 @@ Ref<RTCRtpTransformableFrame> LibWebRTCRtpTransformableFrame::clone()
     return adoptRef(*new LibWebRTCRtpTransformableFrame(WTFMove(rtcClone), m_isAudio));
 }
 
+void LibWebRTCRtpTransformableFrame::setOptions(const RTCEncodedAudioFrameMetadata& metadata)
+{
+    ASSERT(m_isAudio);
+    // FIXME: Support more metadata.
+    if (metadata.rtpTimestamp)
+        m_rtcFrame->SetRTPTimestamp(*metadata.rtpTimestamp);
+}
+
+void LibWebRTCRtpTransformableFrame::setOptions(const RTCEncodedVideoFrameMetadata& newMetadata)
+{
+    ASSERT(!m_isAudio);
+    auto rtcMetadata = static_cast<webrtc::TransformableVideoFrameInterface*>(m_rtcFrame.get())->Metadata();
+
+    if (newMetadata.frameId)
+        rtcMetadata.SetFrameId(*newMetadata.frameId);
```

Comment:
can't this be done in `VideoFrameMetadata` instead? 
e.g. `rtcMetadata.setWith(newMetadata)`